keps/sig-network/2593-multiple-cluster-cidrs/README.md

```diff
@@ -827,14 +935,18 @@ checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
 
+`multicidrset_cidrs_allocations_total` metric with value > 0 will indicate enablement. 
+The operator can also check if ClusterCIDR objects are created, this can be done
+using `kubectl get cc -A`
+
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 We will carry-over existing metrics to the new controller: https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/nodeipam/ipam/cidrset/metrics.go#L26-L68
 
 They are:
--   cidrset_cidrs_allocations_total - Count of total number of CIDR allcoations
--   cidrset_cidrs_releases_total - Count of total number of CIDR releases
--   cidrset_usage_cidrs - Gauge messuring the percentage of the provided CIDRs
+-   multicidrset_cidrs_allocations_total - Count of total number of CIDR allcoations
```

Comment:
In line 944 it says: "We will carry-over...".

Has that happened? If so (hopefully), please update.
If not, please add that to a Beta criteria explicitly.

Reply:
Yes that has been done, changed it to `We have carried-over...`